### PR TITLE
Add cancel endpoints to API specification

### PR DESCRIPTION
**Pull request by jujaga**
# Description

This PR adds the API cancel endpoint to the specification. This endpoint is intended to allow users to be able to cancel queued messages if they are still delayed and waiting for their specified time to send.

This change is required as it closes one of the usage loops that users will need in order to manage queued messages.

[[SHOWCASE-334]](https://apps.nrs.gov.bc.ca/int/jira/browse/SHOWCASE-334)

## Types of changes

Documentation (non-breaking change with enhancements to documentation)

## Checklist

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [x] I have checked that unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have updated the OpenAPI 3.0 `v*.api-spec.yaml` documentation (if appropriate)
- [x] I have added necessary documentation (if appropriate)

## Further comments

A visual representation can be found at the [Redoc Documentation](https://ches-pr-24-9f0fbe-dev.pathfinder.gov.bc.ca/api/v1/docs)

### CODE REVIEW

**Comment by bcgov-nr-csst** [comment]
SonarQube Analysis: https://sonarqube-9f0fbe-tools.pathfinder.gov.bc.ca/dashboard?id=common-hosted-email-service-pr-24

**Comment by bcgov-nr-csst** [comment]
SonarQube Analysis: https://sonarqube-9f0fbe-tools.pathfinder.gov.bc.ca/dashboard?id=common-hosted-email-service-pr-24

File: app/src/docs/v1.api-spec.yaml [hunk 1]
```diff
@@ -229,24 +218,21 @@ paths:
...
-        schema:
-          type: string
-          format: uuid
-          example: 00000000-0000-0000-0000-000000000000
-      - $ref: '#/components/parameters/StatusFilter'
     get:
       summary: Status of a single message
+      description: >-
+        This endpoint retrieves and returns all of the information related to
+        msgId. Unlike the queryable endpoint, this singleton endpoint will yield
```
Comment (loneil) [inline app/src/docs/v1.api-spec.yaml]: wouldn't use the word singleton here

Comment (jujaga) [reply app/src/docs/v1.api-spec.yaml]: Changed to single element

**Pull request merged**